Docs/mc_definition.tex

```diff
@@ -110,8 +110,20 @@ \section{Configuration Management}
 
 Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) "hookup" connections and (2) "physical" connections, discussed below.
 
+To capture antenna status, an {\em a priori} antenna status table has been defined to facilitate a work flow to incorporate new antennas into the array and capture and known recalcitrant antennas.  The table provides an enumerated status for each antenna station.  The status is one of:
+\begin{itemize}\setlength\itemsep{-.3em}
+	\item not\_connected
+	\item needs\_checking
+	\item known\_bad
+	\item passed\_checks
+\end{itemize}
+
+
+\subsection{Operation Workflow}
+The work flow is for the operations team to review recently connected antennas on a roughly monthly basis to determine if any should move to another state.  Initially, all antennas have given a "not\_connected" status.  When they are connected, they will be moved to the "needs\_checking" state.  In operation, at each review an antenna may then be moved into one of the states of:  "needs\_checking", "known\_bad", or "passed\_checks".  Antennas not in one of the states of "not\_connected", "known\_bad", or "passed\_checks" is passed along for processing.  Antennas in one of those three states are called "dubitable" antennas, and for backward compatibility the so-called "dubitable" list may be generated.
```

Comment:
I think this text is wrong: "Antennas not in one of the states of "not\_connected", "known\_bad", or "passed\_checks" is passed along for processing"
I think both "passed_checks" and "needs_checking" are included in the processed data (the latter are included because to check them they need to be in the data)

Reply:
Also, we're no longer calling anything "dubitable"...

Reply:
I forgot to update the mc_def doc.